samples/subsys/edac/README.rst

```diff
@@ -34,7 +34,9 @@ Getting help
 ============
 
 After the application has started help can be read with the following
-command::
+command:
+
+.. code-block:: console
```

Comment:
Is console for Linux console?

Reply:
Strictly speaking yes, it's for bash sessions, you're right. But we can still benefit from it to highlight the prompt.
This also makes the prompt not selectable in the text field.

 The only downside would be that should a command be a match for a bash keyword (probably quite unlikely), it might get highlighted funny.

Note that #64058 is also introducing a fix to the "copy" button, and having syntax highlight helps make it clear that one can expect the prompt to not end up being copied to the clipboard.

Reply:
Maybe it should not be `console` ? Some other block style.

Reply:
I check generated code: https://builds.zephyrproject.io/zephyr/pr/64059/docs/samples/subsys/edac/README.html Probably, looks OK.